flexmeasures/ui/templates/crud/account.html

```diff
@@ -1,183 +1,394 @@
+                    <div class="col-md-3">
+                      <label for="logo_url" class="control-label"
+                        >Logo URL</label
+                      >
+                      <input
+                        type="text"
+                        class="form-control"
+                        id="logo_url"
+                        name="logo_url"
+                        value="{{ account.logo_url or menu_logo }}"
+                      />
+                    </div>
+                  </div>
+
+                  {% if user_is_admin %}
+                  <div class="form-group">
+                    <div class="col-md-3">
+                      <label for="consultant_account_id" class="control-label"
+                        >Consultant Account</label
+                      >
+                      <select
+                        class="form-select"
+                        aria-label="Default select example"
+                        id="consultant_account_id"
+                        name="consultancy_account_id"
+                      >
+                        <option
+                          value="{{ account.consultancy_account_id }}"
+                          selected
+                        >
+                          {% if account.consultancy_account_id %} {% for
+                          consultancy_account in accounts %} {% if
+                          consultancy_account.id ==
+                          account.consultancy_account_id %} {{
+                          consultancy_account.name }} {% endif %} {% endfor %}
+                          {% else %} Select Account {% endif %}
+                        </option>
+                        {% for account in accounts %}
+                        <option value="{{ account.id }}">
+                          {{ account.name }}
+                        </option>
                         {% endfor %}
-                    </tbody>
-                </table>
-            </div>
-            </div>
+                      </select>
+                    </div>
+                  </div>
+                  {% endif %}
+                </div>
+
+                <button
+                  class="btn btn-sm btn-responsive btn-success create-button"
+                  type="submit"
+                  value="Save"
+                  style="
+                    margin-top: 20px;
+                    float: right;
+                    border: 1px solid var(--light-gray);
+                  "
+                >
+                  Save
+                </button>
+              </div>
+            </fieldset>
+          </form>
         </div>
+      </div>
     </div>
+    <div class="col-md-8">
+      <div
+        id="toastcontainer"
+        class="toast align-items-center text-bg-primary border-0"
+        role="alert"
+        aria-live="assertive"
+        aria-atomic="true"
+      >
+        <div class="d-flex">
+          <div class="toast-body" id="toastbody"></div>
```

Comment:
I like [the design with a header](https://getbootstrap.com/docs/5.0/components/toasts/#stacking). Maybe the color of the square in the title could then be red for errors but the primary (account) color for normal info messages.

Reply:
I will work on that 👍

Reply:
I have implemented this 👌